### DISCUSSION TITLE
Screen Drop Outs on Mac Studio

### DISCUSSION THREAD

**Discussion by bradleyandrew** [MonitorControl/MonitorControl#1013] [Q&A]
Hello Everyone,

I hope you are well!

I've got a Mac Studio and it's connected to 2x Gigabyte M32U 4K Displays via TB3 -> Dsiplay Port 1.4 Cables. The monitors show up in System Report and I can view them in System Preferences and set them to 144Hz. However from time to time, usually around 1-3 times a day the image will just drop out. It usually happens only on one monitor but sometimes it happens on both. When the drop out occurs the screen goes black for a moment, then the boot screen for the monitor comes up and then it re-connects to macOS and displays correctly.

I've done a bit of reading and it seems it could be related to a high refresh rate issue, well on PC at least and people seem to solve it with the refresh rate wound down to 100Hz. But there isn't much info in regards to Mac, I've set my refresh rates to 98Hz and it still happens quite often, though it does feel like less often.

I do notice that sometimes when I adjust the brightness with MonitorControl that seems to trigger a cut out, it only happens when you reach 100 brightness. I'm thinking about running it for a day or two without Monitor Control to see if that makes a difference.

I have noticed a few times when the drop out happens that someone is using the blender or kettle in the kitchen, I thought maybe it could be power related but it doesn't seem likely. Perhaps just a coincidence as there have been many times when a drop out happens and no such appliance is being used.

Specs for the monitors are here: https://www.displayspecifications.com/en/model/1bb927e3

 If anyone has had any issues like this or could recommend some further troubleshooting steps that would be appreciated.

Thanks!

**Comment by waydabber** [comment]
It might be a cable or power related issue or a problem with the display PSU (100% brightness probably draws more power). I don't think it is software related, if DDC commands would mess with the display's firmware it would happen for all commands probably.

**Reply by bradleyandrew** [reply]
Thanks for that information. It happened on the same monitors when connect using USB-C to USB-C for video on an Intel MacBook Pro too so my suspicion is at the monitors. 

The power cable is standard IEC lead so I don't think that's the issue. As you say, perhaps it's the PSU. 

Good to know it's not DDC. I did try half a day without MonitorControl running and it still happened.
